Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts

```diff
@@ -23,6 +23,7 @@
             $scope.shouldShowCategories = systemCategories.length > 0;
             $scope.system = itSystemUsage.itSystem;
             $scope.lastChangedBy = itSystemUsage.getLastChangedByUserFullName();
+            $scope.lastChanged = moment(itSystemUsage.lastChanged).format(Kitos.Constants.DateFormat.DanishDateFormat);
```

Comment:
use the rendering helper just like in kendo- That one takes all of the edge cases into account